app/src/main/java/com/example/playlistmaker/library/ui/view/LibraryFragment.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.playlistmaker.library.ui.view
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.example.playlistmaker.R
+import com.example.playlistmaker.databinding.FragmentLibraryBinding
+import com.google.android.material.tabs.TabLayoutMediator
+
+class LibraryFragment: Fragment() {
+
+    private var _binding: FragmentLibraryBinding? = null
+    private val binding: FragmentLibraryBinding
+        get() = _binding!!
+    private lateinit var tabLayoutMediator: TabLayoutMediator
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        _binding = FragmentLibraryBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        binding.viewPager.adapter = FragmentLibraryAdapter(childFragmentManager, lifecycle)
```

Comment:
🍏 FragmentLibraryAdapter теперь можно наследовать от той реализации FragmentStateAdapter, которая в конструктор принимает фрагмент, а не fragmentManager.
Тогда здесь достаточно будет сделать так:
```suggestion
        binding.viewPager.adapter = FragmentLibraryAdapter(this)
```